src/esn.calendar.libs/app/components/time-select/time-select.controller.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+angular.module('esn.calendar.libs')
+  .controller('calTimeSelectController', calTimeSelectController);
+
+function calTimeSelectController($scope, calMoment, esnDatetimeService, esnI18nService) {
```

Comment:
Mh... Why don't you do it like before (passing `locale` and `timeFormat` via bindings)? This way the code is tightly coupled to `esnDatetimeService` and `esnI18nService`.